.circleci/config.yml

```diff
@@ -869,7 +869,7 @@ jobs:
   acceptance-kind-1-23-consul-nightly-1-11:
     environment:
       - TEST_RESULTS: /tmp/test-results
-      - CONSUL_IMAGE: "docker.mirror.hashicorp.services/hashicorppreview/consul:1.11-dev"
+      - CONSUL_IMAGE: "docker.mirror.hashicorp.services/hashicorppreview/consul:1.11-dev-ent"
```

Comment:
this repo probably needs to be `consul-enterprise. I see 1.11.3 image, but not the generic 1.11: https://hub.docker.com/layers/consul-enterprise/hashicorppreview/consul-enterprise/1.11.3-ent-dev/images/sha256-065da6fa536ba8572c131d37df5138ea24ed11302549352dce90503339acc74a?context=explore

Reply:
Could we start a test run with those images for now since it takes a while for tests to run?